packages/edit-post/src/components/sidebar/plugin-sidebar/index.js

```diff
@@ -66,7 +67,7 @@ function PluginSidebar( props ) {
 						/>
 					) }
 				</SidebarHeader>
-				<Panel>
+				<Panel className={ className }>
```

Comment:
Hi @nosolosw, I think plugins should be able to apply class names to the sidebars they implement. Given that this panel is not the top level element I feel the className should not be applied here.
What if we apply the className to the sidebar itself:
```
			<Sidebar
				name={ sidebarName }
				label={ __( 'Editor plugins' ) }
				className={ className }
			>
```
This way if a plugin wants to customize the sidebar header it is possible to do so.
Of course, if the plugin wants to customize just the panel it is also possible with `.my-sidebar-test .components-panel`.

The sidebar panel currently does not accepts a className so we would also need to add className support to it.

Reply:
@jorgefilipecosta my feeling is that plugins should some have the flexibility to style their UI _within_ the WordPress design patterns. Adding the class to the sidebar body where the plugin content is going to be rendered makes sense to me because the author can get rid of an extra div just used to customize that body. I don't see how a plugin would benefit from styling the header, though - it might actually be against the unified UI WordPress experience to offer that kind of tweaking?